src/resolvers/Post/index.ts

```diff
@@ -0,0 +1,8 @@
+import { PostResolvers } from "../../types/generatedGraphQLTypes";
+import { comments } from "./comments";
+import { creator } from "./creator";
+
+export const Post: PostResolvers = {
```

Comment:
This isn't wrong but I meant resolver for creator of comment not creator of post. One more thing to notice here is that post and comment whenever they're fetched will probably always need the creator object along with them. So, populating creator each time a comment is fetched might be a good choice.

Reply:
Makes sense. Would implement a `creator` resolver for the `comment` as well.